en/admin/config.rst

```diff
@@ -2174,6 +2178,10 @@ The following are other parameters. The type and value range for each parameter
 
     **tde_default_algorithm** is a parameter that configures the default algorithm used when creating the TDE encryption table. Log and temporary data are always encrypted using the algorithm set with this parameter when they have to be encrypted. **AES** or **ARIA** can be set. For more information on encryption algorithms, refer to :ref:`tde-algorithm`.
 
+**recovery_progress_logging_interval**
+    
+ **recovery_progress_logging_interval** is a parameter to decide whether the details of recovery is printed or not and configure its period. If it is set bigger than 0, the total works and remained works to do of three phase of recovery, Analysis, Redo and Undo are printed. When this is set smaller than 5, it is set to 5.
```

Comment:
It would be nice to explain the unit (second).

Reply:
OK, I have revised.